src/garage/torch/algos/trpo.py

```diff
@@ -46,27 +42,24 @@ def __init__(self,
                  env_spec,
                  policy,
                  baseline,
+                 optimizer=None,
                  max_path_length=100,
-                 policy_lr=3e-4,
                  num_train_per_epoch=1,
                  discount=0.99,
                  gae_lambda=0.98,
-                 max_kl=0.01,
```

Comment:
can you make this `max_kl_step` to match the tf branch? and make sure the rest of the hyperparam names match the TF branch?

if you feel strongly, you can change them in tf instead.

Reply:
I renamed the argument in torch branch.